README.md

```diff
@@ -8,32 +6,8 @@
 [![codecov](https://codecov.io/gh/JuliaDynamics/Agents.jl/branch/main/graph/badge.svg)](https://codecov.io/gh/JuliaDynamics/Agents.jl)
 [![Package Downloads](https://shields.io/endpoint?url=https://pkgs.genieframework.com/api/v1/badge/Agents)](https://pkgs.genieframework.com?packages=Agents)
 
-Agents.jl is a [Julia](https://julialang.org/) framework for agent-based modeling (ABM). All further information about Agents.jl are provided in the docs.
-
-## Contributions
-
-Any contribution to Agents.jl is welcome! For example you can:
-
-  * Add new feature or improve an existing one (plenty to choose from the "Issues" page)
-  * Improve the existing documentation
-  * Add new example ABMs into our existing pool of examples
-  * Report bugs and suggestions in the Issues page
+Agents.jl is a pure [Julia](https://julialang.org/) framework for agent-based modeling (ABM). Agents.jl is [objectively the fastest open source ABM framework](https://github.com/JuliaDynamics/ABM_Framework_Comparisons) out there, routinely being 100x faster than competing open software (MASON, NetLogo, Mesa). Agents.jl is also the only open source ABM framework that fully implements simulations on Open Street Maps.
 
-## Citation
+Besides its incredible performance, two more major highlights of Agents.jl are its simplicity and the total amount of features. Agents.jl is designed to require writing minimal code and being very simple to learn and use, even though it provides an extensive API with thousands of possibilities for agent actions. This simplicity is due to the intuitive space-agnostic modelling approach we have implemented: agent actions are specified using generically named functions that do not depend on the actual space the agents exist in or on the properties of the agents themselves. Overall this leads to ultra fast model prototyping where even changing the space of the agents is matter of only a couple of lines of code.
 
-If you use this package in a publication, please cite the paper below:
-```
-@article{Agents.jl,
-  doi = {10.1177/00375497211068820},
-  url = {https://doi.org/10.1177/00375497211068820},
-  year = {2022},
-  month = jan,
-  publisher = {{SAGE} Publications},
-  pages = {003754972110688},
-  author = {George Datseris and Ali R. Vahdati and Timothy C. DuBois},
-  title = {Agents.jl: a performant and feature-full agent-based modeling software of minimal code complexity},
-  journal = {{SIMULATION}},
-  volume = {0},
-  number = {0},
-}
-```
```

Comment:
I think these two sections are still very important to highlight in the readme, restoring them seems good enough to me

Reply:
Yes for the citation. No for contributing. Surely, someone that wants to contribute should at least visit the online documentation.